### DISCUSSION TITLE
mode simplifier de la timeline

### DISCUSSION THREAD

**Discussion by paulomafio50** [skarab42/marv#129] [Ideas]
des presets pour la time line (fade in fade out .....)

**Comment by mirobane** [comment]
je serait plutôt pour son inverse, style After Effect comme dans ce screen. (translate, rotate, scale et opacity)
![image](https://user-images.githubusercontent.com/23696751/108599613-7fb63880-7392-11eb-811d-5147bc49f7fa.png)
